mirth.rb

```diff
@@ -28,9 +28,9 @@ def add_data
 router = ActionDispatch::Routing::RouteSet.new
 
 router.draw do
-  get '/show/data', to: DailyDataController.action(:show_data)
-  post '/add/data', to: DailyDataController.action(:add_data)
-  match '*path', via: :all, to: DailyDataController.action(:all_paths)
+  get '/show/data', to: 'daily_data#show_data'
+  post '/add/data', to: 'daily_data#add_data'
+  match '*path', via: :all, to: 'daily_data#all_paths'
```

Comment:
Interesting how `match '*path'` doesn't also capture the `via: all` part. I tried to just have 

```match '*path', to: 'daily_data#all_paths'``` 

and this is the error I got: 

```
You should not use the `match` method in your router without specifying an HTTP method. (ArgumentError)
If you want to expose your action to both GET and POST, add `via: [:get, :post]` option.
If you want to expose your action to GET, use `get` in the router:
  Instead of: match "controller#action"
  Do: get "controller#action"
```

I guess it is because the request method =/= request path, and you can technically direct `match '*path'` for _only_ POST requests to one action?